db/migrate/20191029020518_divers.rb

```diff
@@ -0,0 +1,12 @@
+class Divers < ActiveRecord::Migration[5.2]
```

Comment:
you should be using the proper naming conventions! this should be `CreateDivers` otherwise you could get potential naming collisions